### PR TITLE
ASoC: SOF: amd: Add pci revision id check

### PR DESCRIPTION
**Pull request by Venkata-Prasad-Potturu**
Add pci revision id check for renior and rembrandt platforms.

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by vijendarmukunda** [comment]
test this please

File: sound/soc/sof/amd/acp.h [hunk 1]
```diff
@@ -55,6 +55,9 @@
 
 #define ACP_DSP_TO_HOST_IRQ			0x04
 
+#define ACP_RN_PCI_ID				0x01
+#define ACP_RMB_PCI_ID				0x6F
```
Comment (plbossart) [inline sound/soc/sof/amd/acp.h:59]: typo in commit message s/renior/renoir/

Don't ask a French dude to review something that contains the name of a French painter haha.

Comment (Venkata-Prasad-Potturu) [reply sound/soc/sof/amd/acp.h:59]: Okay, Thanks :)
Will correct it.

**Review by plbossart — APPROVED** [review]

**Comment by plbossart** [comment]
test this please

**Review by ranj063 — APPROVED** [review]

**Pull request merged**